terraform/cost_management.tf

```diff
@@ -0,0 +1,44 @@
+data "azurerm_subscription" "current" {
+  count = contains(var.clouds, "azure") ? 1 : 0
+}
+
+resource "azurerm_resource_group" "current" {
+  count    = contains(var.clouds, "azure") ? 1 : 0
+  name     = substr("Stacklet-${var.customer_prefix}-cost", 0, 23)
+  location = var.resource_group_location
+}
+
+resource "azurerm_storage_account" "cost" {
+  count               = contains(var.clouds, "azure") ? 1 : 0
+  name                = local.storage_account_name
+  resource_group_name = azurerm_resource_group.current[0].name
+
+  location                 = azurerm_resource_group.current[0].location
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+}
+
+resource "azurerm_storage_container" "cost" {
+  count                = contains(var.clouds, "azure") ? 1 : 0
+  name                 = "cur"
```

Comment:
I'd like to use "cost" rather than "cur" just because "cur" seems to be AWS-centric.